### PR TITLE
Fix emit 0 count if encounters error

### PR DESCRIPTION
**Pull request by yux0**
**What changed?**
Fix emit 0 count if encounters error

**Why?**
Fix emit 0 count if encounters error

**How did you test it?**

**Potential risks**

### CODE REVIEW

**Review by yycptt — APPROVED** [review]

File: common/domain/replication_queue.go [hunk 1]
```diff
@@ -336,13 +336,14 @@ func (q *replicationQueueImpl) emitDLQSize() {
 				).IncCounter(
 					metrics.DomainReplicationQueueSizeErrorCount,
 				)
+			} else {
+				q.metricsClient.Scope(
```
Comment (mkolodezny) [inline common/domain/replication_queue.go:340]: is there the corresponding metric on the dash?

File: common/domain/replication_queue.go [hunk 2]
```diff
@@ -336,13 +336,14 @@ func (q *replicationQueueImpl) emitDLQSize() {
...
+					metrics.DomainReplicationQueueScope,
+				).UpdateGauge(
+					metrics.DomainReplicationQueueSizeGauge,
+					float64(size),
+				)
 			}
-			q.metricsClient.Scope(
```
Comment (mkolodezny) [inline common/domain/replication_queue.go:340]: nit: lets make sure it wont make the existing alerts going off.

File: common/domain/replication_queue.go (hunk 1 above)
Comment (yux0) [reply common/domain/replication_queue.go:340]: Yes

File: common/domain/replication_queue.go (hunk 2 above)
Comment (yux0) [reply common/domain/replication_queue.go:340]: I updated the alert to use this one.

**Pull request merged**